report/Introduction/IntroductionCtrl.tex

```diff
@@ -1 +1,3 @@
-\input{Introduction/IntroductionIntro.tex}
\ No newline at end of file
+\chapter{Introduction}
```

Comment:
Vi har to chapters der hedder introduction. Kig i Main.tex filen

Reply:
Det er rettet nu, det skal ikke være i main filen